CONTRIBUTING.md

```diff
@@ -85,6 +85,89 @@ the test suite without lint checks:
 This project relies on [Flow] to ensure functions and
 classes are used correctly. Run all Flow checks with `npm run flow-check`.
 
+The first steps to learn how to fix flow errors are:
+
+- learn how to read the flow annotations
+- learn how to write new or change the existent type definitions
```

Comment:
typo: learn how to write new type definitions or change the existing definitions